README.md

```diff
@@ -1 +1,360 @@
 # Centrapay Documentation
+
+## Overview
+
+### Introduction
+
+Welcome to Centrapay! Payments through Centrapay are accomplished via a two-step process, involving both the merchant and consumer.  
+
+The API is designed to support payment terminals, point of sale, shopping cart, vending and unattended transactions.
+
+It enables consumers to pay with a range of digital assets, including bitcoin and vouchers. The merchant can initiate payment from a consumer or the consumer can push a payment to the merchant. It will also be possible to activate our services in the payment flow such as carbon offsetting.
+
+### Payment Flow
+
+#### Payment flow using Pocket Vouchers
+
+```
+requests.create > Pocket Voucher Code Entry screen on terminal > User enters Pocket Voucher Code into terminal > requests.pay > terminal Loading >Webhook Notification > terminal display result.
+```
+
+#### Payment flow using QR Code
+
+```
+requests.create > Generate QR Code on terminal > User selects payment type on phone > Phone Scans > requests.pay > Authenticated and Processed via Phone > Webhook Notification > terminal display result.
+You will need to create a request ‘/requests.create’ first and use the requestId for making a payment ‘/requests.pay’.
+```
```

Comment:
Should be in a flow chart and maybe more like  
request is created > qr code is generated and displayed by the terminal > consumer scans qr code displayed on terminal > User selects payment type from a list of payment options a merchant supports > We verify payment and notify terminal > terminal displays result